<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Curve</class>
 <widget class="QWidget" name="Curve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>543</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Curve</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QWidget" name="rightWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget_curve">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/*设置TabWidget中tab_1的样式*/
#tab_1.QWidget{
	background-color: rgba(108, 117, 125, 65);
}
/*设置TabWidget中tab_2的样式*/
#tab_2.QWidget{
	background-color: rgba(108, 117, 125, 65);
}
/*设置TabWidget中QTabBar的样式*/
QTabBar::tab{
    background-color: #0B0E11;
	/*font-family:Consolas;  */  /*设置tab中的文本的字体*/
	font-size:10pt;
	color:#ced4da;    /*设置tab中的文本的颜色*/
	border-top-left-radius: 5px;    /*设置tab的边框的圆角（左上圆角）*/
	border-top-right-radius: 5px;    /*设置tab的边框的圆角（右上圆角）*/
	min-width: 8px;
	padding: 5px;
}
/*设置TabWidget中QTabBar的tab被选中时的样式*/
QTabBar::tab:selected{
    background-color: rgba(108, 117, 125, 65);
}
 
/*设置TabWidget中鼠标悬浮在QTabBar的tab上，但未选中该Tab的样式*/
QTabBar::tab:hover:!selected {
    background-color: rgba(108, 117, 125, 45);
}
 
/*设置TabWidget的边框的样式*/
QTabWidget::pane {
    border: 1px solid rgba(108, 117, 125, 65);
}
 
/*当打开多个tab，右侧出现，点击后，可以向前向后的按钮的样式*/
QTabBar QToolButton {
    border: none;
	color: rgb(255, 206, 6);
    background-color: #0b0e11;
}
 
QTabBar QToolButton:hover {
	background-color: #161a1e; 
}</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::TabPosition::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::TabShape::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::TextElideMode::ElideNone</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_curve_more" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
background-color: rgba(255, 255, 255, 0);
border:0;
border-bottom:1.5 solid rgb(255,255,255)
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_more">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="1" column="1">
          <widget class="QWSwitcherButton" name="AutoScaleAxisX" native="true">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="Checked" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QtMaterialTextField" name="lineEdit_cacheDuration" native="true">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QWSwitcherButton" name="AutoScaleAxisY" native="true">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="Checked" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QWSwitcherButton" name="qWSwitcherButton_PlotMode" native="true">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="stateSetQuantity" stdset="0">
            <number>3</number>
           </property>
           <property name="selectState" stdset="0">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_curve_base" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QtMaterialRaisedButton" name="curve_addTab" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialRaisedButton" name="curve_clearData" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWSwitcherButton" name="curve_pauseResumeButton" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="stateSetQuantity" stdset="0">
            <number>2</number>
           </property>
           <property name="selectState" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWSwitcherButton" name="curve_moreLess" native="true">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="stateSetQuantity" stdset="0">
            <number>2</number>
           </property>
           <property name="selectState" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWSwitcherButton" name="curve_gridLine" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="stateSetQuantity" stdset="0">
            <number>2</number>
           </property>
           <property name="selectState" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QWidget</extends>
   <header>qt_material_widgets/qtmaterialtextfield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QWidget</extends>
   <header>qt_material_widgets/qtmaterialraisedbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QWSwitcherButton</class>
   <extends>QWidget</extends>
   <header>qwool/qwswitcherbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
